httpx/client.py

```diff
@@ -1105,17 +1188,24 @@ def dispatcher_for_url(self, url: URL) -> AsyncDispatcher:
         stream: bool = False,
         auth: AuthTypes = None,
         allow_redirects: bool = True,
+        retries: typing.Union[RetriesTypes, UnsetType] = UNSET,
         timeout: typing.Union[TimeoutTypes, UnsetType] = UNSET,
     ) -> Response:
         if request.url.scheme not in ("http", "https"):
             raise InvalidURL('URL scheme must be "http" or "https".')
 
         timeout = self.timeout if isinstance(timeout, UnsetType) else Timeout(timeout)
 
+        retries = self.retries if isinstance(retries, UnsetType) else Retries(retries)
+
         auth = self.build_auth(request, auth)
 
-        response = await self.send_handling_redirects(
-            request, auth=auth, timeout=timeout, allow_redirects=allow_redirects,
+        response = await self.send_handling_retries(
+            request,
+            auth=auth,
+            retries=retries,
+            timeout=timeout,
+            allow_redirects=allow_redirects,
```

Comment:
Something that we *could?* choose to do would be something like...

```python
if retries is not None:
    response = await self.send_handling_retries(...)
else:
    response = await self.send_handling_redirects(...)
```

Reply:
Did this a bit differently because it's easier to have `retries` always be a `Retries` instance (i.e. `httpx.Retries()` means "retries disabled" the same way `httpx.Timeout()` refers to "timeouts disabled"). So I added this switch inside `.send_handling_redirects()` to make the flow clearer when retries are disabled.